packages/modules/src/modules/banners/designableBanner/components/DesignableBannerReminderSignedOut.tsx

```diff
@@ -75,7 +71,7 @@ function Signup({
 
             <form css={styles.form} onSubmit={handleSubmit(() => onSubmit(email))}>
                 <TextInput
-                    label="Email address"
+                    label=""
```

Comment:
is this accessible if there's no label?

Reply:
https://guardian.github.io/storybooks/?path=/docs/source-react-components_textinput--docs
there's a `hideLabel` prop to visually hide it while keeping the label text in the HTML

Reply:
Excellent suggestion, thank you